[package]
authors = ["Solana Maintainers <maintainers@solana.foundation>"]
edition = "2018"
name = "solana-accountsdb-plugin-manager"
description = "The Solana AccountsDb plugin manager."
version = "1.8.14"
repository = "https://github.com/solana-labs/solana"
license = "Apache-2.0"
homepage = "https://solana.com/"
documentation = "https://docs.rs/solana-validator"

[dependencies]
bs58 = "0.4.0"
crossbeam-channel = "0.4"
libloading = "0.7.0"
log = "0.4.11"
serde = "1.0.130"
serde_derive = "1.0.103"
serde_json = "1.0.67"
solana-accountsdb-plugin-interface = { git = "https://github.com/solindex/accountsdb-plugin-interface.git", version = "=1.8.14" }
solana-logger = { git = "https://github.com/solindex/logger.git", version = "=1.8.14" }
solana-measure = { git = "https://github.com/solindex/measure.git", version = "=1.8.14" }
solana-metrics = { git = "https://github.com/solindex/metrics.git", version = "=1.8.14" }
solana-rpc = { git = "https://github.com/solindex/rpc.git", version = "=1.8.14" }
solana-runtime = { git = "https://github.com/solindex/runtime.git", version = "=1.8.14" }
solana-sdk = { git = "https://github.com/solindex/sdk.git", version = "=1.8.14" }
thiserror = "1.0.21"

[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]
